AdyenEncryption/JOSE/Encryption Algorithms/JSON Web Encryption/JSONWebEncryptionGenerator.swift

```diff
@@ -0,0 +1,50 @@
+//
+// Copyright (c) 2021 Adyen N.V.
+//
+// This file is open source and available under the MIT license. See the LICENSE file for more info.
+//
+
+import Foundation
+
+internal protocol AnyJSONWebEncryptionGenerator {
+    func generate(withPayload: Data,
+                  publicRSAKey: SecKey,
+                  mockInitializationVector: Data?,
+                  mockContentEncryptionKey: Data?,
```

Comment:
not a big fan of `mockXXX` property on init AND protocol 🤔

How about make 2 inits (one "official" and one for testing)?

Reply:
removed it, since its not needed for now